libs/editor/src/main/java/org/wordpress/android/editor/EditorMediaUploadListener.java

```diff
@@ -9,4 +9,5 @@ public interface EditorMediaUploadListener {
     void onMediaUploadProgress(String localId, float progress);
     void onMediaUploadFailed(String localId, String errorType);
     void onGalleryMediaUploadSucceeded(long galleryId, long remoteId, int remaining);
+    void onMediaUploadPaused(String toString);
```

Comment:
This is an odd parameter name, presumably a typo. 
```suggestion
    void onMediaUploadPaused(String localId);
```